app/views/hyrax/dashboard/collections/_modal_remove_sub_collection.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="modal fade" id="collection-remove-sub-collection" tabindex="-1" role="dialog">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <form class="delete-collection-form">
+        <div class="modal-body">
+          <p>Removing this sub-collection will not remove it from the repository, only from this parent colleciton.  Are you sure you want to remove this sub-collection?</p>
```

Comment:
Typo: "only from this parent colleciton"
Should be: "only from this parent **collection**"

Can you move the string definition to hyrax.en and use I18n here?